src/Render/Render.jl

```diff
@@ -7,4 +7,33 @@ Render
 
 using Compat
 
+import ..Elements: Node
+
+
+immutable EmptyRenderError <: Exception
+    msg :: AbstractString
+end
+
+function checkchildren(n::Node)
+    isempty(n.children) && throw(EmptyRenderError(string("'$(typeof(n))': '",
+                                haskey(n.data, :title) ?  n.data[:title] : n.data[:outname],
+                                "' -> has no content")))
```

Comment:
Do we need `->` here. It always makes me immediately think of anonymous functions, which clearly isn't what the error message is about. Just

```
"' has no content."))
```

would be fine.